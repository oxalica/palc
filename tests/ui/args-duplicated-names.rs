#[derive(palc::Args)]
struct Short {
    #[arg(short)]
    hello: i32,
    #[arg(short)]
    hell: i32,

    #[arg(long)]
    long1: i32,
    #[arg(long = "long1")]
    long2: i32,
}

fn main() {}
