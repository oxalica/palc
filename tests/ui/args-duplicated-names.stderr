error: duplicated argument names
 --> tests/ui/args-duplicated-names.rs:6:5
  |
6 |     hell: i32,
  |     ^^^^

error: previously defined here
 --> tests/ui/args-duplicated-names.rs:4:5
  |
4 |     hello: i32,
  |     ^^^^^

error: duplicated argument names
  --> tests/ui/args-duplicated-names.rs:10:18
   |
10 |     #[arg(long = "long1")]
   |                  ^^^^^^^

error: previously defined here
 --> tests/ui/args-duplicated-names.rs:9:5
  |
9 |     long1: i32,
  |     ^^^^^
